src/store/mutations.js

```diff
@@ -160,7 +157,7 @@ function orderBookSnapshotReceived(state, res) {
   const orderBook = dex.formOrderBook(res.asks, res.bids);
   orderBook.pair = res.pair;
 
-  state.orderBook = orderBook;
+  Vue.set(state, 'orderBook', orderBook);
```

Comment:
Curious why this change?

Reply:
It was in desktop this way. Does this allow Vue to pick up that it changed better?